CHANGELOG.md

```diff
@@ -2,33 +2,50 @@
 
 Full documentation for MIGraphX is available at [MIGraphX Documentation](https://rocmdocs.amd.com/projects/AMDMIGraphX/en/latest/).
 
-## MIGraphX 2.5 for ROCm 5.5.0
+## MIGraphX 2.7 for ROCm 5.7.0
```

Comment:
I would assume a CHANGELOG would be appended not replaced for each version. Like how rocm-cmake does the CHANGELOG:
https://github.com/RadeonOpenCompute/rocm-cmake/blob/develop/CHANGELOG.md

Reply:
Yup makes sense.